fbpic/boundaries/boundary_communicator.py

```diff
@@ -198,6 +243,9 @@ def divide_into_domain( self, zmin, zmax, p_zmin, p_zmax ):
         self.iz_start_procs = [ k*Nz_per_proc for k in range(self.size) ]
         # Get the enlarged number of cells in each domain (includes guards)
         self.Nz_enlarged_procs = [ n+2*self.n_guard for n in Nz_domain_procs ]
+        # Add damping region to first and last domain
+        self.Nz_enlarged_procs[0] += self.n_damp
+        self.Nz_enlarged_procs[-1] += self.n_damp
```

Comment:
Note: at some point, we should check that this does not introduce load imbalance between the GPUs, i.e. maybe we could determine Nz_per_proc once we added the damp cells.